### ISSUE TITLE
Compilation error in OMEdit/OMEditLIB/OMC/OMCProxy.cpp

### ISSUE THREAD

**Issue by anotheruserofgithub** [OpenModelica/OpenModelica#9387]
### Description

I encounter a compilation error on master branch after a complete clean, due to [the introduction of storeAST/restoreAST](https://github.com/OpenModelica/OpenModelica/commit/6aebcc480315f042544a851929852f91f54ce8ba#diff-90ae47ecdb6770e1a4c25bbdd05b6f846151f5293d4a68c920cd34846d69c619) in OMEdit.
I don't know if I'm missing something or if this is really preventing full builds for about a month now.
Jenkins seems to have reported the related commit as buggy but unfortunately I get a 404 error when I try to see [the details](https://test.openmodelica.org/jenkins/job/OpenModelica/job/master/3824/display/redirect).
For the time being I simply return dummy values from the failing functions in order to complete the build, since I don't use/need them.

### Steps to Reproduce

```
git clean -fdx
git submodule foreach --recursive git clean -fdx
git reset --hard
git submodule foreach --recursive git reset --hard
git submodule update --init --recursive
autoconf
./configure
make
```

### Expected Behavior

Compilation ends successfully.

### Version and OS

 - OpenModelica Version: master branch
 - OS: Ubuntu Linux 64-bit

**Comment by casella** [comment]
@adeas31 can you please comment on that?

**Comment by adeas31** [comment]
Need to clean and build the compiler again. You have an old `OpenModelicaScriptingAPIQt.h` and `OpenModelicaScriptingAPIQt.cpp`.

**Comment by anotheruserofgithub** [comment]
I thought `git clean -fdx` would already take care of that. Well, it does indeed, but in fact I found out the problem is because the build process is not using the locally compiled omc but the package-installed omc. After removing the omc package (and the ones that depend on it) everything compiles fine.

Here is the command that compiles OpenModelicaScriptingAPI with omc package uninstalled, as expected:
```
/home/user/OpenModelica/build/bin/omc -g=MetaModelica OpenModelicaScriptingAPI.mos
```
And here with omc package installed (even in single-threaded `make`, and I can confirm there exists no `./build/bin/omc` when this command is executed):
```
/usr/bin/omc -d=failtrace -g=MetaModelica OpenModelicaScriptingAPI.mos
```
So omc is not compiled beforehand because it is already found in `/usr/bin`.

Would it be possible to force using the locally compiled omc even if there is already one in the system?

**Comment by perost** [comment]
> Would it be possible to force using the locally compiled omc even if there is already one in the system?

Yes, `./configure --without-omc` to force bootstrapping of the compiler or `./configure --with-omc=/path/to/omc` if you want to use a specific `omc` for the build.
